src/main/java/com/onarandombox/multiverseinventories/YamlWorldGroupManager.java

```diff
@@ -30,19 +30,21 @@
 
     private final CommentedYamlConfiguration groupsConfig;
 
-    YamlWorldGroupManager(final MultiverseInventories inventories, final File groupConfigFile, final Configuration config)
-            throws IOException {
+    YamlWorldGroupManager(final MultiverseInventories inventories, final Configuration config) throws IOException {
         super(inventories);
 
-        // Check if the group config file exists.  If not, create it and migrate group data.
+        // Check if the group config file exists. If not, create it and migrate group data.
+        File groupsConfigFile = new File(plugin.getDataFolder(), "groups.yml");
+        boolean groupsConfigFileExists = groupsConfigFile.exists();
         boolean migrateGroups = false;
-        if (!groupConfigFile.exists()) {
+        if (!groupsConfigFile.exists()) {
             Logging.fine("Created groups file.");
-            groupConfigFile.createNewFile();
+            groupsConfigFile.createNewFile();
             migrateGroups = true;
         }
         // Load the configuration file into memory
-        groupsConfig = new CommentedYamlConfiguration(groupConfigFile, true);
+        boolean supportsCommentsNatively = Integer.parseInt(this.plugin.getServer().getBukkitVersion().split("\\.")[1]) > 17;
```

Comment:
Generally, i think @dumptruckman dont really prefer version string check to apply different behaviour, but I think in this case likely the easiest soln.

Reply:
I'm still getting the same message in my console:


[14:23:53] [Server thread/ERROR]: Error occurred while enabling Multiverse-Inventories v4.2.3-SNAPSHOT-bUNKNOWN (Is it up to date?)
java.lang.StackOverflowError: null
	at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingValueComment.produce(ParserImpl.java:684) ~[snakeyaml-1.30.jar:?]



Followed by another 1,023 lines of the same thing. My server then disables Multiverse-Inventories.

Is the fix meant to be just a copy and paste the jar situation?


Thanks for your hard work and all the devs! Appreciate the support

Reply:
This PR fixes the issue of comment duplication. But it doesn't remove the comments that were already added.

You'll need to open the two config files and remove the duplicate comments to get it running again.

Reply:
> Generally, i think @dumptruckman dont really prefer version string check to apply different behaviour, but I think in this case likely the easiest soln.

I'm not particularly fond of it either, but I didn't find any other way to do it. Open to suggestions.

Reply:
> This PR fixes the issue of comment duplication. But it doesn't remove the comments that were already added.
> 
> You'll need to open the two config files and remove the duplicate comments to get it running again.

Thanks for that! I tried just deleting the entry, and it came up with the same error, so I just deleted the two config files and started again and it's working fine. Cheers!

Reply:
yea, that will be a better option. PaperLib  should be a small dependency